src/nat/agent/rewoo_agent/register.py

```
@@ -158,10 +158,7 @@ async def _response_fn(input_message: ChatRequest) -> ChatResponse:
 
         except Exception as ex:
             logger.exception("ReWOO Agent failed with exception: %s", ex)
```

Ruff (0.13.1)

Redundant exception object included in `logging.exception` call

(TRY401)